File: pgm/ABC_R/ABC_212_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
zero <- 0
Gold <- "Gold"
Silver <- "Silver"
Alloy <- "Alloy"
A <- strsplit(arr[1], split = " ")[[1]][1] |>
    as.integer()
B <- strsplit(arr[1], split = " ")[[1]][2] |>
    as.integer()
if (A > zero && B > zero) {
    Alloy |>
        cat("\n", sep = "")
} else if (A == zero) {
    Silver |>
        cat("\n", sep = "")
} else {
    Gold |>
        cat("\n", sep = "")
}
q("no")

File: pgm/ABC_Dart/ABC_212_A.dart
// import 'dart:collection';
// import 'dart:convert';
import 'dart:io';
// import 'dart:math';
// import 'dart:typed_data';

void main() {
  final [int A, int B] = stdin
      .readLineSync()!
      .split(' ')
      .map(int.parse)
      .toList();
  const String Gold = 'Gold';
  const String Silver = 'Silver';
  const String Alloy = 'Alloy';
  const int zero = 0;
  print(
    A > zero && B > zero
        ? Alloy
        : A == zero
        ? Silver
        : Gold,
  );
  exit(0);
}
